### PR TITLE
Minor Fixes for Semantic Hub and Registry

### PR DESCRIPTION
**Pull request by JaneMcBrain**
1. Adds Depricated Status to SE Hub Filter: https://jira.catena-x.net/browse/A1SLDT-673
2. Removes Divider between Key-Value Props 
3. Removes Divider between Twin endpoint-values
4. Changes order of items in twin detail

### CODE REVIEW

File: cx-portal/src/components/pages/SemanticHub/ModelTable.tsx [hunk 1]
```diff
@@ -60,6 +60,10 @@ const ModelTable = ({ onModelSelect }: ModelTableProps) => {
           value: 'DRAFT',
           label: 'Draft',
         },
+        {
+          value: 'DEPRECATED',
+          label: 'Deprecated',
```
Comment (oyo) [inline cx-portal/src/components/pages/SemanticHub/ModelTable.tsx:65]: no i18n here?

**Review by oyo — APPROVED** [review]
see remark about i18n - otherwise okay

**Review by PI-FerasAhmad — APPROVED** [review]
looks good to me

File: cx-portal/src/components/pages/SemanticHub/ModelTable.tsx (hunk 1 above)
Comment (JaneMcBrain) [reply cx-portal/src/components/pages/SemanticHub/ModelTable.tsx:65]: Good point - but this Status is shown always in English, so I thought it is better to see the values, that will be shown in the table...

Comment (oyo) [reply cx-portal/src/components/pages/SemanticHub/ModelTable.tsx:65]: In cases like that we could just see the status as key and in the translation files add keys like `content.semantichub.table.values.status.DEPRECATED=veraltet` and then in the column definition use a `valueGetter: (row) => t(<getKeyForValue>)` for status...

But in this case I guess all the keys are understood or even commonly used in german so it's hardly worth the effort.

Comment (JaneMcBrain) [reply cx-portal/src/components/pages/SemanticHub/ModelTable.tsx:65]: Do you have an example of this in the code already? I don't know, if I understood it correctly...

Comment (oyo) [reply cx-portal/src/components/pages/SemanticHub/ModelTable.tsx:65]: afaik we don't have this implemented anywhere. But never mind - ignore my comments for now. The status values are okay to understand in other languages. Maybe in the future there are other pages where it makes more sense to implement that. Consider the PR approved as it is :)

**Pull request merged**